lib/utils/pyhelp.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * This file is part of the Micro Python project, http://micropython.org/
+ *
+ * The MIT License (MIT)
+ *
+ * Copyright (c) 2016 Colin Hogben
```

Comment:
It's a line of code defining an API.  Agreed, it's hardly the most ground-breaking piece of code ever written, but it didn't exist before, and I wrote it, so (subject to jurisdiction, IANAL) I de facto own the copyright.  I didn't see anything about copyright in the contributor guidelines, but I'm happy to assign copyright to Damien or yourself or to the project as a whole if that is preferred.